git_reqs/importtools.py

```diff
@@ -37,6 +40,59 @@ def import_from_markdown(reqmodule, req_md):
                 req['Description'] = req['Description'] + md_line
             md_line = md_file.readline()
 
+def parse_requirement_links(reqmodule, source_root):
```

Comment:
This function is very large and kind of messy. Could you clean it up and divide it into smaller functions?